pyfr/solvers/navstokes/kernels/bcs/slp-adia-wall.mako

```diff
@@ -5,7 +5,7 @@
 <%include file='pyfr.solvers.navstokes.kernels.bcs.common'/>
 <%include file='pyfr.solvers.navstokes.kernels.flux'/>
 
-<%pyfr:macro name='bc_ldg_state' params='ul, nl, ur'>
+<%pyfr:macro name='bc_rsolve_state' params='ul, nl, ur'>
```

Comment:
Should these changes be part of their own PR?

Reply:
I don't think so, it's a minor fix that doesn't change anything except making the BC consistent between Euler/NS

Reply:
Break it out just to make the history cleaner (as everything here will likely be squashed into a single EF commit, and seeing that come up when  doing a git blame on a BC file might be confusing.